src/style.css

```
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  flex-direction: row;
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars :nth-child(n+1):not(:last-child) {
+  margin: 0 4px 0 0;
+}
```

```suggestion
.stars__star :not(:last-child) {
  margin-right: 4px;
}
```
This spelling would be much clearer.
Better to use shorthand margin property in case of 2 and more directions.
If you modify only one direction of margin - use property only for that direction.

---

thank you